.travis.yml

```diff
@@ -3,6 +3,10 @@ sudo: false
 notifications:
   email: false
 
+branches:
+  only:
+    - master
```

Comment:
should this be `main`?

Reply:
nvm, you’re too fast

Reply:
lmao just caught that myself. it's updated!